yb-voyager/cmd/analyzeSchema.go

```diff
@@ -455,16 +455,26 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			summaryMap["FUNCTION"].invalidCount++
 		} else if regMatch := partitionColumnsRegex.FindStringSubmatch(sqlInfo.stmt); regMatch != nil {
 			allColumns := strings.Trim(regMatch[3], "() ")
-			allColumnsList := strings.Split(allColumns, "(")
-			primaryKeyColumns := allColumnsList[len(allColumnsList)-1]
+			allColumnsList := utils.CsvStringToSlice(allColumns)// List of all columns with PK declaration at the last position
```

Comment:
For the comments in this entire section, it might be helpful to provide an example in the comments itself to visualise the extraction process better, no need to write entire sentences explaining the actual workflow. For eg., you can just provide an expected format of what `allColumnsList` would look like, and cascade this example as a comment down towards each variable you're using

Reply:
added two examples showing two cases.